templates/boot-fastboot/generic-fastboot-boot-template.jinja2

```diff
@@ -0,0 +1,85 @@
+{%- extends 'base/kernel-ci-base.jinja2' %}
+{%- block protocols %}
+protocols:
+  lava-lxc:
+    name: lxc-{{ platform }}-art-conv
+    template: debian
+    distribution: debian
+    release: jessie
+    mirror: http://mirror.bytemark.co.uk/debian
+{%- endblock %}
+
+{% block actions %}
+actions:
+{%- block deploy_lxc %}
+
+  - deploy:
+      namespace: tlxc
```

Comment:
What does the "t" in `tlxc` mean?   I think this would be clearer to just use "host" and "target" for namespaces.

Reply:
oky